mpmp/src/view/Frame.java

```diff
@@ -1,133 +1,242 @@
+
+    public Frame(Model m) {
+        this.m = m;
+        chatDisp = new ChatDisp();
+        playerDisp = new PlayerDisp();
+        createFrame();
+        setVisible(true);
+    }
+
+    /**
+     * Create the frame.
+     */
+    public void createFrame() {
+        setTitle("MPMP");
+        setResizable(true);
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        gameboard = new JSVGCanvas();
+        gameboard.setBackground(new Color(0, 0, 0, 0));
+
+        try {
+            setContentPane(new JLabel(new ImageIcon(ImageIO.read(new File("graphics/background.png")))));
+            gameboard.setFont(Font.createFont(Font.TRUETYPE_FONT, this.getClass().getResourceAsStream("/graphics/font/SourceSansPro-Light.ttf")));
+
+            String parser = XMLResourceDescriptor.getXMLParserClassName();
+            SAXSVGDocumentFactory f = new SAXSVGDocumentFactory(parser);
+            String uri = new File("graphics/svg/gameboard.svg").toURI().toString();
+            org.w3c.dom.Document doc = f.createDocument(uri);
+            gameboard.setDocument(doc);
+
+        } catch (FontFormatException | IOException e) {
+            //XXX Handle exception
+        }
+
+        gameboard.addGVTTreeRendererListener(new GVTTreeRendererAdapter() {
+            public void gvtRenderingPrepare(GVTTreeRendererEvent e) {
+                setTitle("MPMP - Loading...");
+            }
+
+            public void gvtRenderingCompleted(GVTTreeRendererEvent e) {
+                setTitle("MPMP");
+                SVGDocument svgd = gameboard.getSVGDocument();
+                final Element element = svgd.getElementById("");
+                System.out.println(gameboard.getX() + "  " + gameboard.getSize() + "  " + gameboard.getRenderRect() + "  " + gameboard.getSVGDocument().getRootElement());
+                Graphics g = gameboard.getGraphics();
+                g.drawOval(0, 0, 50, 50);
+
+                gameboard.invalidate();
+            }
+        });
+
+        //Set all layouts
+        setLayout(new BorderLayout());
+        left = new JPanel();
+        chat = new JPanel();
+        bottom = new JPanel();
+        bottomMenu = new JPanel();
+        
+        left.setLayout(new BoxLayout(left, BoxLayout.Y_AXIS));
+        chat.setLayout(new BoxLayout(chat, BoxLayout.Y_AXIS));
+        bottom.setLayout(new BoxLayout(bottom, BoxLayout.X_AXIS));
+        bottomMenu.setLayout(new BoxLayout(left, BoxLayout.X_AXIS));
+
+        //Set testing Buttons
+        JButton player1 = new JButton("Player 1");
+        JButton player2 = new JButton("Player 2");
+        JButton player3=  new JButton("Player 3");
+        JButton player4 = new JButton("Player 4");
+        JButton player5 = new JButton("Player 5");
+        JButton player6 = new JButton("Player 6");
+        JButton trade = new JButton("Tauschen");
+        JButton buyHouse = new JButton("Haus kaufen");
+        JButton buyPlot = new JButton("Strasse kaufen");
```

Comment:
Should be `Straße`. I know this is just for testing, but we don't want to forget that.